internal/controllers/network/tests/network-update/01-updated-resource.yaml

```
@@ -0,0 +1,14 @@
+---
+apiVersion: openstack.k-orc.cloud/v1alpha1
+kind: Network
+metadata:
+  name: network-update
+spec:
```

changing the admin state needs admin credentials.

You need to add:
```suggestion
spec:
  cloudCredentialsRef:
    cloudName: openstack-admin
    secretName: openstack-clouds
```

---

Then in that case, it's not needed here, as the resource already has a reference on the admin secret.